openff/toolkit/topology/mm_molecule.py

```diff
@@ -0,0 +1,475 @@
+"""
```

Comment:
Let's prepend this filename with an underscore to ensure that it is very private :-)

Reply:
👍  5833f2058d92f1dd72b7ae02b7586b67a8cb8aa2 / 67474c741f75adbce08d556a925edb5cde3493d2